README.md

```diff
@@ -47,8 +47,45 @@ The app targets shoppers across Australia who are looking for unique and quality
 ![Application-Architecture-Diagram](./docs/images/application-architecture-diagram/application_architecture_diagram.png)
 
 ## User Stories 
+We constructed user stories and personas to improve the online store's user experience for Japanese food products. The process involves identifying and embodying customers' unique needs and expectations, including a system administrator at Sakura company.
 
-Working in progress...
+### Initial Market Research
+1. As a busy parent, I want search functionality with a category filtering option to save time searching for the right product.
+
+2. As a busy parent, I want a contact form 24/7 to quickly ask questions or queries about products and services in the online shop, regardless of time or location.
+
+3. As a user who often changes residence, I need to easily update my profile to ensure accurate delivery despite frequent changes in residence.
+
+4. As a user, I want the convenience of a Product Search Bar that allows me to find specific pantry items effortlessly. This will save me time and ensure that I quickly locate the products I'm looking for.
```

Comment:
Maybe change `I want the convenience of a Product Search Bar`

to `I want to be able to search products by keywords, which allows me to ......`